### DISCUSSION TITLE
cryptic sentence on documentation for display BI7 7

### DISCUSSION THREAD

**Discussion by ouinouin** [arendst/Tasmota#20483] [Support request]
on the doc : https://tasmota.github.io/docs/Displays/#descriptor-file

on the section explaining the I:  to initialize display
Bi7 7 on the number of arguments set indicate a wait of 150 ms. On I2C all hex values are sent to I2C.
What means BI7 7 ?  i suppose its a typo meaning BIT7 , bu t in this case i dont understand well what does mean something like 
```
11,80
29,80
```
the register 11 is written with zeroes and a 150ms delay  ? 
same for the register 29 ? 
the wait is before the writing or after the writing ? 

thanks for clarifications

**Comment by sfromis** [comment]
Yes, I think that "Bi7 7" is a typo for "Bit 7".

Bit 7 is when seeing a 8-bit number (0-255, or hex 0x00-0xFF) as 7 bits, numbered 76543210. Bit 7 alone (with the others being zero would be `0x80`. As the text says that all values are in hex, the number 3 for 3 arguments would therefore become 83 when setting bit 7.

This means that a random line like `E8,3,85,00,78` becomes `E8,83,85,00,78` when setting bit 7 on the number giving the argument count.

To be clear, I'm only posting from my general knowledge about how computers and terminology works, I have no specific experience with doing things in that `:I` section. Thus I'm not 100% certain.

I'm a little bit surprised by seeing hex mentioned, and still having one-digit values. While this is perfectly valid, let me (for context) add that in decimal the value 3 would have 128 added for bit 7, thus becoming 131.
